crates/orderbook/src/arguments.rs

```diff
@@ -101,14 +101,10 @@ pub struct Arguments {
     #[clap(long, env, parse(try_from_str), default_value = "false")]
     pub enable_presign_orders: bool,
 
-    /// If solvable orders haven't been successfully update in this time in seconds attempting
+    /// If solvable orders haven't been successfully updated in this many blocks attempting
     /// to get them errors and our liveness check fails.
-    #[clap(
-        long,
-        default_value = "300",
-        parse(try_from_str = shared::arguments::duration_from_seconds),
-    )]
-    pub solvable_orders_max_update_age: Duration,
+    #[clap(long, default_value = "24")]
+    pub solvable_orders_max_update_age_blocks: u64,
```

Comment:
This is used by Orderbook to determine whether the auction is out of date by long enough that something probably went wrong with the updating process. I'm changing it to a block number instead of duration because the block number is already included in the auction information that is stored in the database while the time is not. Both values measure passage of time so it shouldn't matter.